modules/datagrid/72/launch/added/launch/infinispan-config.sh

```diff
@@ -282,16 +282,40 @@ function configure_cache() {
        </encoding>"
   fi
 
-  if [ -n "$(find_env "${prefix}_CACHE_EVICTION_STRATEGY")$(find_env "${prefix}_CACHE_EVICTION_MAX_ENTRIES")" ]; then
-    if [ -n "$(find_env "${prefix}_CACHE_EVICTION_STRATEGY")" ]; then
-      local CACHE_EVICTION_STRATEGY="strategy=\"$(find_env "${prefix}_CACHE_EVICTION_STRATEGY")\""
-    fi
+  if [ -n "$(find_env "${prefix}_CACHE_MEMORY_EVICTION_SIZE")$(find_env "${prefix}_CACHE_EVICTION_MAX_ENTRIES")" ]; then
+
+    local evictionSize="$(find_env "${prefix}_CACHE_MEMORY_EVICTION_SIZE")"
     if [ -n "$(find_env "${prefix}_CACHE_EVICTION_MAX_ENTRIES")" ]; then
-      local CACHE_EVICTION_MAX_ENTRIES="size=\"$(find_env "${prefix}_CACHE_EVICTION_MAX_ENTRIES")\""
+      log_warning "Environment variable <cacheName>_CACHE_EVICTION_MAX_ENTRIES has been deprecated. Use <cacheName>_CACHE_MEMORY_EVICTION_SIZE instead."
```

Comment:
it would be better if instead of `<cacheName>` you substitute in the actual cache name, i.e. `${prefix}`